newrelic/core/environment.py

```diff
@@ -170,6 +170,20 @@ def environment_settings():
         if hasattr(uvicorn, "__version__"):
             dispatcher.append(("Dispatcher Version", uvicorn.__version__))
 
+    if not dispatcher and "hypercorn" in sys.modules:
+        dispatcher.append(("Dispatcher", "hypercorn"))
+        hypercorn = sys.modules["hypercorn"]
+
+        if hasattr(hypercorn, "__version__"):
+            dispatcher.append(("Dispatcher Version", hypercorn.__version__))
+        else:
+            try:
+                from importlib.metadata import version
```

Comment:
I know this is an agent so sometimes we violate certain best practices (in this particular case importing at run time) that but I think that can be avoided here by referencing sys.modules["importlib"].metadata.version instead of importing it.

Reply:
Moved things around and used the same get_version callable as the sys.modules code.